app/views/income_additional/edit.html.erb

```diff
@@ -13,7 +13,7 @@
     <%= form_for @step, as: :step, builder: MbFormBuilder, url: current_path, method: :put do |f| %>
 
       <% @income_fields.each do |method, label| %>
-        <%= f.mb_money_field method, label, notes: ["Monthly amount"] %>
+        <%= f.mb_input_field method, label, notes: ["Monthly amount"] %>
```

Comment:
I'll take a look at how GCF does this and will see about using that approach instead of the js lib I previously included.

Reply:
cool! ✔️